### DISCUSSION TITLE
Add two quarto websites to gallery

### DISCUSSION THREAD

**Discussion by linogaliana** [quarto-dev/quarto-cli#4157] [Show and Tell]
Hi ! I would like to propose two additional websites for the quarto gallery (https://quarto.org/docs/gallery/):

- Book section: [`utilitR` documentation](https://www.book.utilitr.org/)
- Website section: my [Python for data science course](https://pythonds.linogaliana.fr/)

**Comment by cscheid** [comment]
Can you make a PR on quarto-web? Thanks.

**Reply by linogaliana** [reply]
Yes of course ! Thanks 😺
